packages/apollo-storybook-vue/src/index.js

```diff
@@ -0,0 +1,44 @@
+import createClient from 'apollo-storybook-core';
+import VueApollo from 'vue-apollo';
+
+export default function initializeApollo({
+  typeDefs,
+  mocks,
+  apolloLinkOptions = {},
+  apolloClientOptions = {},
+  typeResolvers,
+  context = {},
+  rootValue = {},
+  // cacheOptions is a necessary config parameter because some use cases will require a pre-configured
+  // fragmentMatcher such as IntrospectionFragmentMatcher, etc.
+  cacheOptions = {},
+  Vue,
+}) {
+  const graphqlClient = createClient({
+    mocks,
+    apolloLinkOptions,
+    apolloClientOptions,
+    typeResolvers,
+    typeDefs,
+    rootValue,
+    context,
+    cacheOptions,
+  });
+
+  // Install the vue plugin
+  Vue.use(VueApollo);
+
+  const apolloProvider = new VueApollo({
```

Comment:
`apolloProvider` will need to be attached to the root Vue instance. I'm still considering how to do this.

Reply:
Not works `Vue.extend` ?
In `storybook-router`, seems that it's wrapped VueRouter with it.
https://github.com/gvaldambrini/storybook-router/blob/master/src/vue.js#L69-L80

Reply:
havent tried it! will do!

Reply:
@kazupon i tried this and got this error:

```
[Vue warn]: Error in beforeCreate hook: "TypeError: Cannot read property 'defaultOptions' of undefined"

found in

---> <WrappedComponent>
       <Anonymous>
         <Root>
```

Reply:
It would be nice if vue-apollo used provide/inject instead of a global mixin which checks for the existance of apolloProvider on $root.

Reply:
Seems vue-apollo uses provide/inject now (https://github.com/Akryum/vue-apollo/releases/tag/v3.0.0-beta.1)

That should make this doable

Reply:
AWESOME!!